models/Airport.ts

```diff
@@ -0,0 +1,16 @@
+export class Airport {
```

Comment:
Thus, if our Airport has more than 100 props, it is assumed that each of them will be written in the constructor, right? It doesn't make any sense at all.

First, do we really need a class? Will we be creating Airport objects ourselves, and therefore need to know what props the object has? I have doubts. 

Just an interface will be much more suitable for our case, and even if we create this object, define more than 5 props through the constructor is a wrong idea.

Also, do we really need to describe each prop? Maybe just Interface { [key: string]: string } would be more suitable for us? 

Second, don't you think it looks strange? Modifier of access only in front of the first arg. 
```typescript
public id: string,
name: string,
city: string
```